docs/api.rst

```diff
@@ -5580,6 +5580,7 @@ Exporting History Events
 
    .. _history_export_schema_section:
 
+   :reqjson bool match_exact_events: If set to true only the events that match exactly the filter will be exported otherwise the whole group of the events that match the filter will be exported.
```

Comment:
```suggestion
   :reqjson bool match_exact_events: If set to true only the events that match the filter exactly will be exported otherwise the whole group of events that match the filter will be exported.
```